src/styles/components/_drawer-course.scss

```diff
@@ -82,6 +82,19 @@
   &__link-title {
     font-size: px-to-rem(14px);
     line-height: 1.5;
+
+    &::after {
+      display: inline-block;
+      height: 0.5em;
+      margin-left: 0.6em;
+    }
+
+    &[data-badge='new']::after {
+      color: $WEB_PRIMARY_COLOR;
+      content: 'NEW';
```

Comment:
Could we use https://developer.mozilla.org/en-US/docs/Web/CSS/attr() to set the content here? Maybe the markup would look like:

```
<a data-new data-new-label="{{ 'i18n.courses.new' | i18n(locale) }}">
```

Reply:
done